test/Compiler/simple/AnonymousImport-1.agda

```diff
@@ -0,0 +1,7 @@
+module AnonymousImport-1 where
```

Comment:
The place for a test case that should fail in the scope or type checker would be `test/Fail` (rather than `test/Compiler/simple`).